subprojects/docs/src/samples/userguide/initScripts/customLogger/kotlin/customLogger.init.gradle.kts

```diff
@@ -0,0 +1,17 @@
+useLogger(CustomEventLogger())
+
+class CustomEventLogger() : BuildAdapter(), TaskExecutionListener {
+
+    override fun beforeExecute(task: Task) {
+        println("[${task.name}]")
+    }
+
+    override fun afterExecute(task: Task, state: TaskState) {
+        println()
+    }
+
+    override fun buildFinished(result: BuildResult) {
+        println("build completed")
+        result.failure?.printStackTrace()
```

Comment:
❓ Couldn't the corresponding Groovy sample use the same syntax here: `result.failure?.printStackTrace()`?

This would have the benefit to let them have the same number of lines, preventing a layout change when switching DSL languages.

BTW, the `public` modifier on overridden methods in the Groovy sample are unnecessary and could be omitted.

Reply:
Yeah, will change. Note that the page should not jump when changing DSLs anymore. If you still see this, please send me details so I can fix it.

Reply:
It's not jumping anymore since your fix, but the layout around change, nothing we can do about it except make samples the same length when it makes sense.